src/SFML/Window/Android/VideoModeImpl.cpp

```diff
@@ -31,19 +31,17 @@
 #include <SFML/Window/VideoModeImpl.hpp>
 
 #include <mutex>
+#include "SFML/Window/VideoMode.hpp"
 
 namespace sf::priv
 {
 ////////////////////////////////////////////////////////////
 std::vector<VideoMode> VideoModeImpl::getFullscreenModes()
 {
-    VideoMode desktop = getDesktopMode();
+    const VideoMode desktop = getDesktopMode();
 
     // Return both portrait and landscape resolutions
-    std::vector<VideoMode> modes;
-    modes.push_back(desktop);
-    modes.emplace_back(Vector2u(desktop.size.y, desktop.size.x), desktop.bitsPerPixel);
-    return modes;
+    return {desktop, VideoMode(Vector2u(desktop.size.y, desktop.size.x), desktop.bitsPerPixel)};
```

Comment:
I hope we can enable the [`modernize-braced-return-init`](https://clang.llvm.org/extra/clang-tidy/checks/modernize/return-braced-init-list.html) clang-tidy check to enforce this style of construction at the `return` statement.